tutorials/multi-modal/image_classification_tutorial.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>need to prompt for an openai key, otherwise notebook fails</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4441/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>